packages/vue-dot/src/templates/MaintenancePage/MaintenancePage.vue

```diff
@@ -14,6 +14,8 @@
 					<p class="mb-0">
 						{{ message }}
 					</p>
+
+					<slot name="content" />
```

Comment:
```suggestion
					<slot />
```
Je pense qu'on peut utiliser le slot par défaut plutôt qu'un slot nommé, ça sera plus simple à l'utilisation et plus cohérent avec les autres cxomposants